training/distributed_training/pytorch/data_parallel/yolov5/yolov5.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>Style - use page title as link text, rather than raw link. I.e. <code>https://docs.aws.amazon.com/fsx/latest/LustreGuide/create-fs-linked-data-repo.html</code> -&gt;  <code>What is Amazon FSx for Lustre?</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>